tests/Maker/AdminMakerTest.php

```diff
@@ -38,45 +38,27 @@
  */
 final class AdminMakerTest extends TestCase
 {
-    /**
-     * @var string
-     */
-    private $projectDirectory;
+    private string $projectDirectory;
 
     /**
      * @var array<string, ModelManagerInterface<object>>
      */
-    private $modelManagers = [];
+    private array $modelManagers = [];
 
-    /**
-     * @var InputInterface
-     */
-    private $input;
+    private ?InputInterface $input = null;
 
     /**
      * @var OutputInterface
      */
     private $output;
```

Comment:
This one didnt got typed.

Reply:
maybe all these "issues" should be reported to rector repo

Reply:
The property is not really needed, removed.